mmv1/products/identityplatform/Config.yaml

```diff
@@ -228,20 +229,29 @@ properties:
       - name: 'signUpQuotaConfig'
         type: NestedObject
         description: |
-          Quota for the Signup endpoint, if overwritten. Signup quota is measured in sign ups per project per hour per IP.
+          Quota for the Signup endpoint, if overwritten. Signup quota is measured in sign ups per project per hour per IP. None of quota, startTime, or quotaDuration can be skipped.
         properties:
           - name: 'quota'
             type: Integer
             description: |
-              A sign up APIs quota that customers can override temporarily.
+              A sign up APIs quota that customers can override temporarily. Value can be in between 1 and 1000.
+            required_with:
```

Comment:
Using this means it's valid to specify `sign_up_quota_configuration {}` to empty or zero the value, is that what's intended? We could just make all these `required` if that is not valid to write (required-ness is scoped to the parent field for nested fields)

Reply:
They current TF surface and the API allow an entirely empty (but present) signUpQuotaConfig. Although it probably doesn't do anything different from removing the entire signUpQuotaConfig block, we don't want to break users who are using it this way. Perhaps this is a good candidate for the next major version? @pashanka

Reply:
Thanks! Seems reasonable to decide out of band but a decision is not merge blocking since we want the current behaviour.

Reply:
Yes, that is correct. Ideally we can make everything as required but that changes from the existing behaviour that empty value works today. Hence, decided to do this to keep it this way for backward compatibility. I can look at fixing this for the next major version.